client/api.go

```diff
@@ -1050,7 +1051,277 @@ func (c *Client) TestSendAnonVotes(
 	return votingElapsedTime, nil
 }
 
-func (c *Client) CreateProcess(oracle *ethereum.SignKeys,
+func (c *Client) CreateAccount(signer *ethereum.SignKeys, infoURI string, nonce uint32) error {
```

Comment:
Please document at least all exported Methods and Types :)

Reply:
yep I did this for all methods i this branch, I can add it on top of this PR, also I simplified the CreateAccount / SetAccountInfoURI and I merged both functions. I just wanted to not add more stuff. Here is the branch with other changes I found useful https://github.com/vocdoni/vocdoni-node/tree/feature/improve_client